src/Redux/State/Board.spec.js

```diff
@@ -1,34 +1,64 @@
 import {
-  addCard,
-  receiveCard,
+  setBlankCardTitle,
   INITIAL_STATE,
-  default as reducer
+  default as reducer,
+  toggleBlankCard,
+  saveCard,
+  removeBlankCard
 } from './Board'
 
 describe('Redux :: Module :: Board', () => {
   it('reduces to initial state', () => {
     expect(reducer()).toEqual(INITIAL_STATE)
   })
 
-  it('reduces the add card action', () => {
+  it('reduces set blank card title action', () => {
     expect(
-      reducer(INITIAL_STATE, addCard()),
+      reducer(INITIAL_STATE, setBlankCardTitle('test blank card title')),
     ).toEqual({
       ...INITIAL_STATE,
-      isLoading: true
+      blankCardTitle: 'test blank card title'
     })
   })
 
-  it('reduces receive card action', () => {
-    const s1 = reducer(INITIAL_STATE, addCard())
-    const s2 = reducer(s1, receiveCard({ title: 'My title' }))
+  it('reduces toggle blank card action', () => {
+    expect(
+      reducer(INITIAL_STATE, toggleBlankCard()),
+    ).toEqual({
+      ...INITIAL_STATE,
+      isBlankCardVisible: true
+    })
```

Comment:
A test should be added to assert that, calling again toggleBlankCard will re-set isBlankCardVisible to false